src/executors/IntervalExecutor.js

```diff
@@ -4,6 +4,19 @@ import {getArgNames} from '../util/function';
 import {buildArgumentsFromRingaEvent} from '../util/executors';
 
 class IntervalExecutor extends ExecutorAbstract {
+  //-----------------------------------
+  // Constructor
+  //-----------------------------------
+  /**
+   * Constructor for a new IntervalExecutor.
+   *
+   * @param thread The parent thread that owns this command.
+   * @param condition The interval keeps looping so long as this returns truthy
```

Comment:
A more technically accurate way to say this might be: "Interval is halted when this callback returns falsy."